packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -399,6 +399,20 @@ describe('shallow', () => {
       expect(wrapper.find('.foo').type()).to.equal('input');
     });
 
+    it('should strip out any non-hostNode', () => {
+      const Foo = props => <div {...props} />;
+      const wrapper = shallow(
+        <div>
+          <Foo className="foo" />
+          <div className="foo" />
+        </div>,
+      );
+      const hostNodes = wrapper.find('.foo').hostNodes();
+      expect(wrapper.find('.foo')).to.have.length(2);
```

Comment:
maybe try `lengthOf` instead of `length` here?

Reply:
Other tests in the same page use `.length`, why so?

Reply:
They should all be using lengthOf; it was just a thought about fixing the travis tests.

Reply:
It doesn't seem to help